common/fixation_entry.go

```diff
@@ -256,7 +277,7 @@ func (fs *FixationStore) getUnmarshaledEntryForBlock(ctx sdk.Context, safeIndex
 		var entry types.Entry
 		fs.cdc.MustUnmarshal(iterator.Value(), &entry)
 
-		if entry.GetBlock() <= block {
+		if entry.Block <= block {
```

Comment:
Question: why did you change from ".GetBlock()" to ".Block"? I always use getters to avoid nil dereference. Isn't it a better practice to always use getters?